### DISCUSSION TITLE
Use browser extension omnibox to search for links

### DISCUSSION THREAD

**Discussion by smth** [WebCrateApp/webcrate#56] [Ideas]
I'd like to use WebCrate as a replacement for browser bookmarks, so a feature I'd like to see is the ability to go directly to links in my WebCrate, from the browser extension. Ideally I think this would be via the omnibox. I was actually expecting this to be what happens when you type "wc" and hit tab - whatever you type after that would be a search of your WebCrate, and pressing Enter / Return would take you to the highlighted entry's link (not to the WebCrate entry). This feels in keeping with the typical function of the address bar.

**Comment by BetaHuhn** [comment]
I like your idea! The omnibox is currently only used to add links, but as you said, it could be used for a lot more. Searching for links in your WebCrate without having to open the app would save a lot of time.

Will add it to the list of planned features, although I can't say when I'll be able to work on it. It shouldn't take much effort though, so maybe I can implement it soon.

**Comment by smth** [comment]
Nice :) This might also be an answer for people [wanting a direct link without going via the WebCrate details](https://github.com/WebCrateApp/webcrate/discussions/51), without compromising the aims of the core app.
